tests/framework/extensions/provisioning/creates.go

```diff
@@ -472,8 +472,8 @@ func CreateProvisioningRKE1AirgapCustomCluster(client *rancher.Client, clustersC
 	}
 
 	logrus.Infof("Register Custom Cluster Through Corral")
-	for numNodes, roles := range rolesPerNode {
-		err = corral.UpdateCorralConfig("node_count", fmt.Sprint(numNodes))
+	for _, roles := range rolesPerNode {
+		err = corral.UpdateCorralConfig("node_count", fmt.Sprint(1))
```

Comment:
is this just always 1?

Reply:
Yes it will create a single node and then register, and continue with different node configurations. Make sense?

Reply:
I could modify the for loop above that you wrote, so account for node. quantity as well.

Reply:
I made the update please check it out!

Reply:
looks great!